File: Modul 6/C/PRAK603-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>

int main (){
    int angka1, angka2, i;
    scanf("%d %d", &angka1, &angka2);
    int matriks1[angka1], matriks2[angka2];

    if (angka1 != angka2) {
        printf ("Jumlah tidak sama") ;
    }
    else {
        for (i = 0; i < angka1; i++) {
            scanf ("%d", &matriks1[i]);
        }
        for (i = 0; i < angka2; i++) {
            scanf ("%d", &matriks2[i]) ;
        }
        for (i = 0; i < angka1; i++) {
            printf ("%d ", matriks1[i] * matriks2[i]) ;
        }
    }
    return 0;
}

File: Modul 6/Python/PRAK603-2310817220025-FirdaKhoirunisa.py
angka1, angka2 = map(int, input().split())
matriks1 = []
matriks2 = []

if angka1 != angka2:
    print("Angka tidak sama")
else:
    baris1 = list(map(int, input().split()))
    baris2 = list(map(int, input().split()))
    for i in range(angka1):
        matriks1.append(baris1[i])
    for i in range(angka2):
        matriks2.append(baris2[i])
    
    for i in range(angka1):
        print(matriks1[i] * matriks2[i], end=" ")